packages/amplify-e2e-tests/src/setup-tests.ts

```diff
@@ -7,3 +7,4 @@ expect.extend({ toBeAS3Bucket });
 const JEST_TIMEOUT = 1000 * 60 * 60; // 1 hour
 
 jest.setTimeout(JEST_TIMEOUT);
+jest.retryTimes(1);
```

Comment:
this change allows a test to retry before all other tests are done, which is ideal because each executor has a worker pool of 3 threads (so there is no need to wait before retrying)